src/main/java/seedu/duke/helper/command/FindPatientCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.duke.helper.command;
+
+import seedu.duke.assets.List;
+import seedu.duke.exception.HalpmiException;
+import seedu.duke.status.Status;
+
+public class FindPatientCommand extends Command {
+
+    public FindPatientCommand(String[] parameterArray) {
+        super(parameterArray);
+    }
+
+    public Status execute(List patientList) throws HalpmiException {
```

Comment:
So the find methods return an array, what do you do with this array?

Reply:
In PatientList/DoctorList/MedicineList, it prints out the filtered Patient/Doctor/Medicine (zero to many) based on the attribute the user wants to filter it by. There are methods in the respective List classes